src/main/scala/ostrich/PrioStreamingTransducer.scala

```diff
@@ -0,0 +1,404 @@
+/*
+ * This file is part of Ostrich, an SMT solver for strings.
+ * Copyright (C) 2020  Matthew Hague, Philipp Ruemmer
```

Comment:
Can you kick out those two guys and add your own name in the new files?

Reply:
OK, fixed